site/en/docs/extensions/migrating/improve-security/index.md

```diff
@@ -29,36 +30,45 @@ There are a few special cases in which executing arbitrary strings is still poss
 
 ## Remove remotely hosted code {: #remove-remote-code }
 
-In Manifest V3, all of your extension's logic must be part of the extension package. You can no longer load and execute remotely hosted files. Examples include:
+In Manifest V3, all of your extension's logic must be part of the extension package. You can no longer load and execute remotely hosted files according to [Chrome Web Store policy](/docs/webstore/program-policies/mv3-requirements/). Examples include:
 
 - JavaScript files pulled from the developer's server.
 - Any library hosted on a [CDN][mdn-cdn].
+- Some third-party libraries.
```

Comment:
I find this a bit unspecified, I'd suggest combining the last two items into:

```
- Bundled third-party libraries that dynamically fetch remote hosted code.
```

Reply:
Done.